### ISSUE TITLE
Support multiple addresses in `contract-addresses.json`

### ISSUE THREAD

**Issue by SebastienGllmt** [PaimaStudios/paima-engine#240]
`contract-addresses.json` is used to store the address of all the contracts you've deployed.

However, right now it stores at most a single address per contract type (as seen in `addAddress`)). This isn't sufficient for games that require deploying multiple contracts of the same type.

**Comment by SebastienGllmt** [comment]
No longer needed after #277
